controllers/feeds/getFeedsV2.js

```diff
@@ -249,6 +257,15 @@ module.exports = async (req, res) => {
         });
       }
     }
+    console.log('DEBUG ', postActors);
+    // get karma score for each post actor
+    const karmaScores = await UsersFunction.getUsersKarmaScore(User, postActors);
+    console.log('DEBUG ', karmaScores);
+    for (let i = 0; i < data.length; i++) {
+      const user = karmaScores.find((user) => user.user_id === data[i].actor.id);
+      data[i].karma_score = user?.karma_score || 0;
+      data[i] = await modifyAnonimityPost(data[i], isBlurredPost);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Remove the debug console logs before merging the code to the main branch. Debugging statements should not be present in production code.
```diff
- console.log('DEBUG ', postActors);
- const karmaScores = await UsersFunction.getUsersKarmaScore(User, postActors);
- console.log('DEBUG ', karmaScores);
+ const karmaScores = await UsersFunction.getUsersKarmaScore(User, postActors);
```
Also, consider adding error handling around the `getUsersKarmaScore` call. If it fails for any reason, it could cause the entire function to fail.


<!-- This is an auto-generated comment by OSS CodeRabbit -->